### DISCUSSION TITLE
Interesting repository

### DISCUSSION THREAD

**Discussion by gentlejob** [X01XX/Unorthodox-Expert-System#4] [General]
Oddly synchronistic with some research I've been doing. Any examples of how you're using this system as a way to bootstrap my intuition and understanding?

I've been thinking of coordinate spaces and differences between entities in a determinstic addressing space that is also able to evolve and measure difference between all other things forming an axiom of identity such that any thing is precisely not every other thing, and then a measure of distance based on signal processing. Trying to discern if there is applicable a universal representation at least at a low level for all things. E.g fourier transform for any "thing" as the method to form the identity.

**Comment by X01XX** [comment]
This is purely a demonstration program.  Actions that cause simple, repeating, changes to the current state, can be mapped out.  At some point, enough is known to able able to use that knowledge to take actions to change the current state.  Initially, the changes desired are simply to insure the action changes are repeatable, then to clarify the understanding.  Later, preloaded states can be sought, or avoided.  The changes mapped out are very simple, low level, but maybe its a start.  In Philosophy, there is a line of thought  that our minds start as a clean slate, then experience is used to learn, changing something in our brain/mind, then we seek and avoid things.
